jetbrains-rider/build.gradle.kts

```diff
@@ -29,9 +30,9 @@ val ideProfile = IdeVersions.ideProfile(project)
 
 plugins {
     id("toolkit-kotlin-conventions")
-    id("toolkit-intellij-subplugin")
     id("toolkit-testing")
     id("toolkit-integration-testing")
+    id("toolkit-intellij-subplugin")
```

Comment:
subplugin?

Reply:
holds conventions for how we setup the `intellij {}` block for IC/IU/RD/GW/etc